src/components/molecules/CheckboxField/__snapshots__/CheckboxField.test.tsx.snap

```diff
@@ -129,13 +135,18 @@ exports[`<CheckboxField /> renders the component with no control icon and no a11
   border-color: #D4D7D9;
 }
 
+.c1:disabled:before {
+  background-color: #FFFFFF;
+}
+
 .c1:checked + label {
   border-color: #009E8E;
   background-color: #ECFCF9;
-  box-shadow: none;
+  box-shadow: none #009E8E;
```

Comment:
This one seems like it'll have an effect on zopa 🤔 ?

Reply:
if its `none` + a colour, isnt that still none?

Reply:
yeah its still a none, its a combined string so to keep theme easier I left it like that. not ideal, I know...